README.md

```diff
@@ -55,7 +55,11 @@ Example:
 node {
     def slackResponse = slackSend(channel: "cool-threads", message: "Here is the primary message")
     slackSend(channel: slackResponse.threadId, message: "Thread reply #1")
-    slackSend(channel: slackResponse.threadId, message: "Thread reply #2")
+    slackSend(
+        channel: slackResponse.threadId,
+        replyBroadcast: true,
```

Comment:
can you separate this out into a different example?
this shouldn't be the default

Reply:
I updated the documentation. What do you think?